docs/release-notes/march-2024.md

```diff
@@ -0,0 +1,119 @@
+  * [<i class="fas fa-external-link fa-xs"></i>1.67.0](https://github.com/semgrep/semgrep/releases/tag/v1.67.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+### Changed
+
+### Fixed
+
+- Visual Studio Code extension: fixed issue where rules weren't downloaded, leading to no scan results.
+
+## 💻 Code
+
+### Added
+
+- Added support for Python's `yield` keyword, enabling the detection of taint findings from taint sources returned by `yield`.
+- Added ability for users to copy file paths displayed in Semgrep Cloud Platform's **Findings** page if they aren't links.
+- Added the ability for users to see if there's a version of a rule they're currently using that supports interfile analysis.
+- **API**: added ability to get rules metadata from the API.
+
+### Changed
+
+- Code analysis started by logged-in users running `semgrep ci` now includes cross-file (interfile) analysis.
+- `.phtml` files are now processed as PHP files and analyzed using PHP rules.
+- Updated PR comments to include links to specific findings in Semgrep Cloud Platform.
+- Users can see all projects, even if they don't have any identified findings, in Semgrep Cloud Platform.
+- Semgrep Code now distinguishes between findings resolved by rule changes and findings resolved due to code modifications. This change applies only to new findings.
+  - Only findings fixed due to code modifications are marked as fixed.
+    - The fix rate calculated by Semgrep Code now includes only such findings.
+  - Findings fixed due to rule changes are marked as resolved.
+- **CLI**: Semgrep clones the repository into the current working directory instead of a `tmp` folder when using the `-- remote` flag.
+
+### Fixed
+
+- Kotlin: Fixed a parsing error when a newline appears between the class name and the primary constructor.
+- Fixed an issue where autofix on variable definitions could not handle semicolons for Java, C++, C#, Rust, Cairo, Solidity, and Dart.
+- Fixed an issue with autofix application on lines with multi-byte characters.
+- Fixed issue where credentials were inadvertently included in a project URL when publishing a custom rule using `semgrep publish`. Running `semgrep publish` generates a `rule-origin-note`, which includes the project URL in the metadata. When this process occurs in a GitLab CI job, GitLab includes the CI job tokens in the project URL. Semgrep now removes the credential from the metadata.
+- Fixed an issue where reachability rules were deleted from Semgrep Registry.
+- Fixed an issue where the timestamp on the findings didn't correspond to the timestamp used by the filter; now, both use the `relevant_since` filter, which provides information about when findings were last reopened.
+
+## ⛓️ Supply Chain
+
+### Added
+
+- Supply Chain now offers lockfile-only support for Swift projects.
+- Added a tour of Supply Chain features in Semgrep Cloud Platform for first-time users, as well as a tour for returning users.
```

Comment:
I never saw this, even as a returning user :cry: . No action, just an observation.

Reply:
Actually, this looks like this isn't GA yet, so maybe we want to remove this?

Reply:
I pulled latest and this was still in, I'll remove it directly on my next commit.